storage/mysql/mysql_test.go

```diff
@@ -28,14 +63,253 @@ func TestMain(m *testing.M) {
 		}
 		klog.Fatalf("Failed to open MySQL test db: %v", err)
 	}
+	defer func() {
+		if err := db.Close(); err != nil {
+			klog.Warningf("Failed to close MySQL database: %v", err)
+		}
+	}()
 	if err := db.PingContext(ctx); err != nil {
 		if *isMySQLTestOptional {
 			klog.Warning("MySQL not available, skipping all MySQL storage tests")
 			return
 		}
 		klog.Fatalf("Failed to ping MySQL test db: %v", err)
 	}
+	testDB = db
+
 	klog.Info("Successfully connected to MySQL test database")
 
+	initDatabaseSchema(ctx)
+
+	noteSigner, err = note.NewSigner(testPrivateKey)
+	if err != nil {
+		klog.Fatalf("Failed to create new signer: %v", err)
+	}
+	noteVerifier, err = note.NewVerifier(testPublicKey)
+	if err != nil {
+		klog.Fatalf("Failed to create new verifier: %v", err)
+	}
+
 	os.Exit(m.Run())
 }
+
+// initDatabaseSchema drops the tables and then imports the schema.
+func initDatabaseSchema(ctx context.Context) {
+	dropTablesSQL := "DROP TABLE IF EXISTS `Checkpoint`, `Subtree`, `TiledLeaves`"
+
+	rawSchema, err := os.ReadFile("schema.sql")
+	if err != nil {
+		klog.Fatalf("Failed to read schema.sql: %v", err)
+	}
+
+	db, err := sql.Open("mysql", *mysqlURI+"?multiStatements=true")
+	if err != nil {
+		klog.Fatalf("Failed to connect to DB: %v", err)
+	}
+	defer func() {
+		if err := db.Close(); err != nil {
+			klog.Warningf("Failed to close db: %v", err)
+		}
+	}()
```

Comment:
This is intended since `multiStatements=true` is only required during the database schema import.

Reply:
I don't really understand what that means, but it sounds important :D 
Worth popping a comment in so that future folks don't try to simplify the code?

Reply:
Good idea. Added some comment.